udtube/data/collators.py

```diff
@@ -6,7 +6,8 @@
 from torch import nn
 import transformers
 
-from . import batches, datasets, indexes
+from .. import special
+from . import batches, datasets
```

Comment:
Minor, but should we consider creating a \_\_init\_\_.py for imports for the data dir? Just to avoid relative imports.

Reply:
[We have one!](https://github.com/CUNY-CL/udtube/blob/master/udtube/data/__init__.py) But it doesn't have an effect in said directory, only in directories higher in the tree. (And I like to import things at module level anyways.)

Reply:
Fair enough. Resolving.